submissions/kissik/data-structures/index.js

```diff
@@ -0,0 +1,127 @@
+const http = require('http');
+
+const fs = require('fs');
+
+const port = process.env.PORT || 3210;
+
+const page404 = {
+  path: 'pages/404.html',
+  type: { 'Content-Type': 'text/html' },
+  verb: 404
+};
+
+const action = {
+  type: { 'Content-Type': 'text/html' },
+  verb: 200
+};
+
+const contentType = {
+  '/': {
+    path: 'pages/index.html',
+    type: { 'Content-Type': 'text/html' },
+    verb: 200
+  },
+  '/css/theme.css': {
+    path: 'css/theme.css',
+    type: { 'Content-Type': 'text/css' },
+    verb: 200
+  },
+  '/img/binoculars.jpg': {
```

Comment:
`'/img/binoculars.jpg'` - repeated several times. Maybe better to put this strings into constants? 🤔 

As well as response codes.

Reply:
strictly speaking they are not the same: the first occurrence has slash symbol prefix:
```
  '/img/binoculars.jpg': {
    path: 'img/binoculars.jpg',    
```
does substring in path look clearer?

Reply:
If you like, you can even map it using filename extension. Like `jpg`, or `css`, or else. Content-type is based on this part of the information, not on the path itself.

But `/${path}` is also an appropriate way to do this.